spec/requests/mutations/sign_up_spec.rb

```diff
@@ -123,7 +123,7 @@
     end
 
     it 'works without the confirmable module' do
-      expect { post_request }.to change(Guest, :count).from(0).to(1)
```

Comment:
I don't think `by` is an improvement over the explicit start and end count you expect